docs/reference/transform/ecommerce-tutorial.asciidoc

```diff
@@ -300,6 +447,13 @@ POST _transform/_preview
 --------------------------------------------------
 // TEST[skip:set up sample data]
 ====
+
+TIP: If the destination index does not exist, it is created the first time you 
+start your {transform}. Unlike pivot {transforms}, however, latest {transforms} 
+do not deduce mapping definitions when they create the index. Instead, they use 
+dynamic mappings. To use explicit mappings, create the destination index 
```

Comment:
Please mention index template.

Reply:
I think in the context of the tutorial, it is better to describe the simplest case, whereas in the API we can cover the if-then-else details.